src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/YFlowExtended.groovy

```diff
@@ -328,4 +341,51 @@ class YFlowExtended {
 
         return new YFlowBidirectionalExam(forward1, reverse1, forward2, reverse2)
     }
+
+    /**
+     * This check allows us to perform main Y-Flow properties comparison after updating operations
+     * Note, some fields such as subFlows.timeUpdate, subFlows.status, timeUpdate, and status are excluded for verification.
+     * @param expectedYFlowExtended
+     * @param isYPointVerificationIncluded
+     */
+    void hasTheSamePropertiesAsExpected(YFlowExtended expectedYFlowExtended, boolean isYPointVerificationIncluded = true) {
```

Comment:
I'd rename to 'hasTheSamePropertiesAs()`, so it could be read as an english sentence in use: 
`yFlow1.hasTheSamePropertiesAs(yFlow2)`